discovery-provider/src/utils/redis_connection.py

```diff
@@ -0,0 +1,12 @@
+"""
+Interface for using a redis connection
+"""
+
+from redis import Redis
+from src.utils.config import shared_config
+
+redis_url = shared_config["redis"]["url"]
+redis = Redis.from_url(url=redis_url)
+
+def get_redis():
```

Comment:
so it looks like this is test-only but might work within the discovery provider - should we use this in `__init__.py` too?

Reply:
yes, we should use it everywhere! i just didn't want to touch everywhere just yet. there are quite a few other places